### PR TITLE
Update RPI Unit Master to fit with Arduino Sensor Unit(s)

### PR DESCRIPTION
**Pull request by dpirjan**
A PoC for sending task(s)

Signed-off-by: Daniel Pirjan <dpirjan@gmail.com>

### CODE REVIEW

File: meta-rpi-home/recipes-piHome/piUnitMaster/files/UnitMasterRPI.c [hunk 1]
```diff
@@ -23,169 +23,393 @@ RaspberryPI versions 1, 2, 3 are pins compatible for pins no 01 .. 26
...
+#define ID_BatteryLevel     0x00080000UL    // (bit_19) Read battery voltage on Analog
+
+//Actuators IDs <actuatorIDs>
+#define ID_Siren            0x00100000UL    // (bit_20)
+#define ID_Buzzer           0x00200000UL    // (bit_21)
+#define ID_Relay            0x00400000UL    // (bit_22)
+#define ID_DoorBell         0x00800000UL    // (bit_23)
+#define ID_IR               0x01000000UL    // (bit_24)
+#define ID_Pump             0x02000000UL    // (bit_25) Water pump
+
```
Comment (aurelian17) [inline meta-rpi-home/recipes-piHome/piUnitMaster/files/UnitMasterRPI.c]: DoorLock = 0x4000000 // bolt or whatever else used for locking the door
please add this too

File: meta-rpi-home/recipes-piHome/piUnitMaster/files/UnitMasterRPI.c [hunk 2]
```diff
@@ -23,169 +23,393 @@ RaspberryPI versions 1, 2, 3 are pins compatible for pins no 01 .. 26
...
+#define ID_GET_TASK         0x80000000UL    
+
+// Default value to wake-up [ms] (watchdog timestamp)
+#define WDT_DT              4000
+
+// Master node for RF mesh
```
Comment (aurelian17) [inline meta-rpi-home/recipes-piHome/piUnitMaster/files/UnitMasterRPI.c]: This is no longer used ..

File: meta-rpi-home/recipes-piHome/piUnitMaster/files/UnitMasterRPI.c [hunk 3]
```diff
@@ -54,56 +54,63 @@ using namespace std;
...
- *  - CNF_MSK : From Master to Unit configure mask value:
+ *              mesh.write msg_type="m" 0x6D
+ *  - CNF_MSK : From Master to Unit, configure mask value:
  *              val = 0 Sensor value isn't delivered (even when interrupt occours)
  *                      Remark:
  *                        When forced, the sensor value is sent to Master, means if
  *                        explicitly is requested then this task is prioritar
  *              val = 1 Sensor value is deliver without restriction
- * 
+ *              mesh.write msg_type="M" 0x4D
```
Comment (aurelian17) [inline meta-rpi-home/recipes-piHome/piUnitMaster/files/UnitMasterRPI.c]: what is this? mesh is supposed to be removed?

File: meta-rpi-home/recipes-piHome/piUnitMaster/files/UnitMasterRPI.c [hunk 4]
```diff
@@ -54,56 +54,63 @@ using namespace std;
...
  *  -SEND_VAL : From Unit to Master, respond with the sensor/actuator value
  *              mesh.write msg_type="V" 0x56
  *  -SEND_ERR : From Unit to Master, return SUCCESS or ERROR No
- *              mesh.write msg_type="R" 0x52
+ *              mesh.write msg_type="E" 0x45
```
Comment (aurelian17) [inline meta-rpi-home/recipes-piHome/piUnitMaster/files/UnitMasterRPI.c]: same here

Comment (dpirjan) [inline meta-rpi-home/recipes-piHome/piUnitMaster/files/UnitMasterRPI.c]: "Copy-paste" and not updated comment after removing mesh lib. Correct description should be "RF24 write". I'm going to replace all "mesh.write" comments.

File: meta-rpi-home/recipes-piHome/piUnitMaster/files/UnitMasterRPI.c (hunk 3 above)
Comment (dpirjan) [inline meta-rpi-home/recipes-piHome/piUnitMaster/files/UnitMasterRPI.c]: Only "unfortunate" comment. Should be RF24 write. I'm going to update it

**Comment by aurelian17** [comment]
Everything looks ok now, merged. Thanks.

**Comment by dpirjan** [comment]
UPS sorry, I saw too late that you close it. Latest PR is with changes we discussed in the morning. Tested only at build without running image on board.

**Comment by aurelian17** [comment]
reopened

**Pull request merged**